### DISCUSSION TITLE
v0.0.17

### DISCUSSION THREAD

**Discussion by mkckr0** [mkckr0/audio-share#37] [Announcements]
v0.0.17  
- If you have downloaded the v0.0.16, you should upgrade to this version right away. The v0.0.16 has no limit of "Loudness Enhancer". If this value that you set too big, it will hurt your ears.
- Add a limit to "Loudness Enhancer". Its max value is `3000mB`.
- Change the step of "Audio Volume" seekbar to `0.01`.  

v0.0.16  
- Add Loudness Enhancer to make the audio louder #34
- Remove "Idle Volume" and "Work Volume". They were implemented by changing system media volume.
- Add "Audio Volume", which is independent from system media volume. Changing it won't change system volume.

<hr /><em>This discussion was created from the release <a href='https://github.com/mkckr0/audio-share/releases/tag/v0.0.17'>v0.0.17</a>.</em>

**Comment by clicky6** [comment]
Wouldn't it be better to completely remove audio volume from audio-share as we can change the system volume directly like we usually do on Android so why to complicate things, What do you think?

**Reply by mkckr0** [reply]
"Audio Volume" provide a way to decrease the volume without changing system volume.

**Comment by ghost** [comment]
What's the quality overall?
I've read README, and FAQ and there is no mention of bitrates, only if I look at the code, but I'm not a coder.
So, what's the quality of it? Will it have Hi-Fi quality later in time? I would donate if it was the case.

**Reply by mkckr0** [reply]
Open server.log and you can see the audio format including sample rate.
```
[2024-02-07 15:54:09.752] [server] [info] AudioFormat:
format_tag: 3
channels: 2
sample_rate: 48000
bits_per_sample: 32
```
As log shown, the sample rate is 48kHz. The sample rate and other parameter are all default values given by Windows Core Audio APIs.

I try to change the audio endpoint's audio format, the sample rate become 192kHz.
![image](https://github.com/mkckr0/audio-share/assets/26296627/90eff36e-c55d-4490-9fad-1d7be60c9968)

**Reply by mkckr0** [reply]
https://github.com/mkckr0/audio-share?tab=readme-ov-file#about-audio-format
